FWCore/Framework/test/test_non_event_ordering_beginLumi_cfg.py

```diff
@@ -14,4 +14,5 @@
                            consumesBeginLuminosityBlock = cms.InputTag("a", "beginLumi"),
                            expectBeginLuminosityBlock = cms.untracked.int32(1))
 
-process.schedule = cms.Schedule(tasks=cms.Task(process.a,process.b,process.c))
```

Comment:
Why require the process to be named?

Reply:
I don't understand your question, could you elaborate?

Reply:
The question should have been "Why require the task to be named?" 
The answer was already given that you had to create a Path to avoid having the modules be removed.